sample/pom.xml

```diff
@@ -40,6 +40,11 @@
       <artifactId>hawkular-accounts-api</artifactId>
       <version>1.0.6-SNAPSHOT</version>
     </dependency>
+    <dependency>
+      <groupId>org.hawkular.accounts</groupId>
+      <artifactId>hawkular-accounts-jaxb-adapters</artifactId>
+      <version>1.0.6-SNAPSHOT</version>
```

Comment:
`${project.version}` ? Well, thinking of it again, is perhaps `versions:set` smart enough to handle the version numbers also here in the dependencies?